### PR TITLE
Fix #119 - Minimal update to fuels: Gas => NaturalGas

### LINKED ISSUE #119: Need update following E+ 9.4.0 fuel type rename
eg: https://github.com/NREL/OpenStudio-workflow-gem/blob/3b5918b6a0d8891e245762600735abe729abe81b/lib/openstudio/workflow/jobs/resources/monthly_report.idf#L36

Edit: seems like monthly_report.idf is unused, and instead this is here:

https://github.com/NREL/OpenStudio-workflow-gem/blob/3b5918b6a0d8891e245762600735abe729abe81b/lib/openstudio/workflow/util/energyplus.rb#L387

This is now "NaturalGas", not "Gas". A few other renames happened. Some fuels were added too. You can find an example of the proper names in with all fuels in IDF form at https://github.com/NREL/OpenStudio/pull/4093/files#diff-ede7c9df11c3641f308134faa451399d and the ruby script used to produced it at https://github.com/NREL/OpenStudio/pull/4093/files#diff-8d6ae98a15673b09352a89e1197100f5

Related: https://github.com/NREL/OpenStudio/pull/4093

### PR DESCRIPTION
**Pull request by jmarrec**
Fix #119 - Minimal update to fuels: Gas => NaturalGas

Fix https://github.com/NREL/OpenStudio/issues/4092

PropaneGas => Propane for eg isn't present, so no need to fix anything. Seems like Gas was the only one.

### CODE REVIEW

**Review by nllong — APPROVED** [review]
These changes look good. We should probably update our test files to be EnergyPlus 9.4 if this is going to be a breaking change.

**Comment by jmarrec** [comment]
@nllong I added a rake task to prepare a LST file (**select** "Text File With List of EnergyPlus Files") that can be run in IDFVersionTranslator to batch update IDF file. and I ran it accross the repo.

image

File: Rakefile [hunk 1]
```diff
@@ -36,3 +36,14 @@ task :test_zip do
...
+desc 'Transition IDF Files'
+task :transition_idf do
+  lst_path = File.join(Dir.pwd, 'IdfFilesToTransition.lst')
+  File.open(lst_path, 'w') do |file|
+    Dir.glob("**/*.idf").each do |idf|
+      file.puts(File.join(Dir.pwd, idf))
+    end
+  end
+  puts "Use the IDFVersionUpdater and point it to #{lst_path}"
+end
```
Comment (jmarrec) [inline Rakefile:49]: `bundle exec rake transition_idf` will create a lst file.

File: .gitignore [hunk 2]
```diff
@@ -33,3 +33,4 @@ run.log
 reports
 */**/out*.osw
 .ruby-version
+IdfFilesToTransition.lst
```
Comment (jmarrec) [inline .gitignore:36]: Ignore the lst file

File: spec/files/results_in_order/measures/XcelEDATariffSelectionandModelSetup/resources/Non-Xcel Gas Firm.idf [hunk 3]
```diff
@@ -1,32 +1,30 @@
+  Version,9.4;
 
-Version,
-  8.1.0.009;                              !- Version Identifier
+  UtilityCost:Tariff,
+    Non - Xcel Gas Firm,     !- Name
+    NaturalGas:Facility,     !- Output Meter Name
```
Comment (jmarrec) [inline spec/files/results_in_order/measures/XcelEDATariffSelectionandModelSetup/resources/Non-Xcel Gas Firm.idf:5]: Transition utility picked up stuff like that

File: Rakefile (hunk 1 above)
Comment (nllong) [reply Rakefile:49]: 👍

**Comment by nllong** [comment]
looks great, thanks for transitioning the idf files!

**Pull request merged**